lib/inspector.rb

```diff
@@ -58,6 +58,14 @@ def info?
     @info
   end
 
+  def display_summary(target = $stdout)
+    padding = { "エラー" => "", "警告" => "  ", "INFO" => "  " }
+    KLASS_TAG.each_value do |value|
+      num = @messages.count{ |msg| msg =~ /^\[#{value}\]/ }
```

Comment:
__[rubocop]__
* Style/SpaceBeforeBlockBraces: Space missing to the left of {.

<!-- Comment by [SideCI](https://sideci.com) -->